mwdb/resources/auth.py

```diff
@@ -236,11 +236,8 @@ def post(self):
         summary: Get password change link for the current user
         description: |
             Requests password change link for currently authenticated user.
-
             Link expires after setting a new password or after 14 days.
-
             Link is sent to the e-mail address set in user's profile.
-
             Requires `manage_profile` capability.
```

Comment:
Hmm.. as I remember these double line breaks are required to have it rendered in separate lines